COLLABORATOR_GUIDE.md

```diff
@@ -24,8 +24,10 @@ understand the project governance model as outlined in
 
 ## Issues and Pull Requests
 
-Courtesy should always be shown to individuals submitting issues and
+Courtesy should **always** be shown to individuals submitting issues and
 pull requests to the Node.js project.
+It is recommanded to be extra welcoming to "first-time-contributors",
```

Comment:
minor nit: blank line before this sentence?

Reply:
Ack